tests/e2e/scripts/capture-events-and-logs.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
```

Comment:
this was generated using AI, its not going to go into production, so minimal effort, does the job